configure.ac

```diff
@@ -6,20 +6,20 @@ dnl # Run ./autogen.sh to build configure script
 dnl #
 dnl ##########################################################################
 
-AC_PREREQ(2.59)
+AC_PREREQ(2.63)
```

Comment:
No, the original was using macros from 2.64 (and has been for a long time). The new macros I use are present in 2.63, but I don't know if they were introduced there. I bumped to 2.63 mainly because:

* 2.64 has been (secretly) required for such a long time, I doubt anyone is using anything older.
* 2.63 is already eight years old.
* At least 2.63 is tested, we don't know about the older ones.
* It doesn't affect tarballs.

All in all I just thought it was a good time to bump. I can document it the way I described in the commit message, or if you think it's a bad idea I can also remove it.